src/AFQMC/config.0.h

```diff
@@ -58,6 +58,16 @@ using SPValueType = SPRealType;
 using ComplexType   = std::complex<RealType>;
 using SPComplexType = std::complex<SPRealType>;
 
+template <class T>
+struct is_complex : std::false_type {};
+
+template <class T>
+struct is_complex<std::complex<T>> : std::true_type {};
```

Comment:
is_complex is already in the codebase
https://github.com/QMCPACK/qmcpack/blob/2db0b25e4ad64b06f11cfb5ba9644c553094571a/src/type_traits/complex_help.hpp#L17